### PR TITLE
Semantic Router uploaded project files

### PR DESCRIPTION
**Pull request by ithanigaikumar**
@hello-fri-end Uploaded all the files used for the Semantic Router application and submitted Pull Request

### CODE REVIEW

**Review by hello-fri-end — CHANGES_REQUESTED** [review]
Hey @ithanigaikumar ! Thank you for making this contribution :muscle: :rocket: . I moved the demo video section to the top in the [template readme](https://github.com/unifyai/demos/blob/main/template/README_TEMPLATE.md) to  better catch the attention of the viewer. Could you please look at the changes and update the README?  You can take a look the [README of LLM-Wars project](https://github.com/unifyai/demos/blob/main/Unify/LLM-Wars/README.md?plain=1) for an example.  Happy to review again once you're done! :)

**Comment by ithanigaikumar** [comment]
@hello-fri-end Thank you I have embedded the video first now

File: Unify/SemanticRouter/README.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# Semantic Router
+[Demo](https://github.com/unifyai/demos/blob/0be2009315c9a11347cf03d02c9dba394d7afd0e/videos/semanticroutervideo.mp4)
```
Comment (hello-fri-end) [inline Unify/SemanticRouter/README.md:2]: You should drag the video from your local storage to the README. Doing so will generate a link that you should put here. Currently, the video isn't embedded into the README. Hope that makes sense :)

File: Unify/SemanticRouter/README.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+<video width="640" height="480" autoplay>
+  <source src="../../../../_static/semanticroutervideo.mp4" type="video/mp4">
+Your browser does not support the video tag.
+</video>
+
+
+
+## Introduction:
+This streamlit application dynamically routes user queries to the most appropriate model based on semantic similarity.User queries are sent to predefined routes of maths or coding or users have the option to create their own custom route. This optimizes output quality .
```
Comment (hello-fri-end) [inline Unify/SemanticRouter/README.md:12]:
```suggestion
This Streamit application dynamically routes user queries to the most appropriate model based on semantic similarity. User queries are sent to predefined routes of maths or coding or users have the option to create their own custom route. This optimizes output quality while minimizing the cost.
```
Can you elaborate this please?

File: Unify/SemanticRouter/README.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+
+
+  
+## Contributors
+
+| Name                          | GitHub Username |
+|-------------------------------|-----------------|
+| Indiradharshini Thanigaikumar | ithanigaikumar  |
+| Jeyabalan Nadar               | jeyabalang      |
```
Comment (hello-fri-end) [inline Unify/SemanticRouter/README.md:46]: Can you convert the user-names to links?

**Review by hello-fri-end — CHANGES_REQUESTED** [review]

File: Unify/SemanticRouter/README.md (hunk 1 above)
Comment (ithanigaikumar) [reply Unify/SemanticRouter/README.md]: @hello-fri-end Hi I've tried doing this but it says my file size is too big,even after compressing it?I am a bit confused by this step : /  I made a youtube video so i could put that in instead?

Comment (hello-fri-end) [reply Unify/SemanticRouter/README.md]: Oh, what's the duration & size in mb of the video you're trying to upload?

Comment (ithanigaikumar) [reply Unify/SemanticRouter/README.md]: @hello-fri-end it was 254,000MB and about 3 minutes long but i put it through the compressor about 3 more times and managed to get a link haha, please could you check if everything is okay to be merged now : )?

File: Unify/SemanticRouter/README.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
+# Semantic Router
+[Demo](https://github.com/ithanigaikumar/demos/assets/107815119/33ceff47-3495-44a9-aad7-c0a3ba3433a8)
+
+<video width="640" height="480" controls>
+  <source src="https://raw.githubusercontent.com/unifyai/demos/0be2009315c9a11347cf03d02c9dba394d7afd0e/videos/semanticroutervideo.mp4" type="video/mp4">
```
Comment (hello-fri-end) [inline Unify/SemanticRouter/README.md:5]: Hey @ithanigaikumar, the `src` here is not correct. Can you take a look at the instructions [here](https://github.com/unifyai/demos/blob/main/template/README_TEMPLATE.md?plain=1#L4-L10)? Here's an example.

**Review by hello-fri-end — CHANGES_REQUESTED** [review]

File: Unify/SemanticRouter/README.md (hunk 4 above)
Comment (ithanigaikumar) [reply Unify/SemanticRouter/README.md]: @hello-fri-end okay I've changed it is it correct now?

**Comment by hello-fri-end** [comment]
LGTM! Thanks for the contribution @ithanigaikumar ! :100:

**Pull request merged**